its/ruling/src/integrationTest/java/org/sonarsource/iac/IacRulingTest.java

```diff
@@ -163,10 +162,10 @@ private void run_ruling_test(String project, Map<String, String> projectProperti
   }
 
   @AfterAll
-  public static void after() {
+  public static void after() throws InterruptedException {
     if (keepSonarqubeRunning) {
       // keep server running, use CTRL-C to stop it
-      new Scanner(System.in).next();
```

Comment:
I couldn't find a way to allow a test run with Gradle to read `System.in`; it leads to exception otherwise.

Reply:
This property is quite old. I wonder why it is affecting you. I introduced it to keep the Orchestrator running when testing locally.